api/swagger.yaml

```
@@ -1356,6 +1356,277 @@ definitions:
           type: "string"
         example: ["/bin/sh", "-c"]
 
+  ImageConfig:
+    description: |
+      Configuration of the image. Fields in this struct are used as defaults
```

Maybe cleaner to avoid "structs" here since that's a Go/language term and not an API thing

```suggestion
      Configuration of the image. These fields are used as defaults
```